project-base/src/Shopsys/ShopBundle/Resources/styles/front/common/components/in/action.less

```diff
@@ -7,22 +7,12 @@
         width: 100%;
         display: flex;
         align-items: center;
-
-        .is-no-flex & {
-            display: block;
-            .clearfix();
-        }
     }
 
     &__left {
         @media @query-md {
+            width: 100%/2;
             order: 1;
-            flex: 1;
```

Comment:
Same as the comment above.

Reply:
Flex: 1 sets all flexible items to be same lenght. Is not necessary bcs, we set calculated width same for both 50%.

Reply:
It is okay for now. We will refactor this later.